kuma/static/styles/components/compat-tables/_bc-signal.scss

```
@@ -463,9 +463,11 @@ div.bc-signal-block {
             transition: border-color .1s;
         }
 
-        &:hover {
+        &:hover:not(.selected) {
             .browser-logo {
                 border-color: #3d7e9a;
```

While you are working on this file, mind changing this to

```suggestion
                border-color: $blue;
```

---

Thanks for the suggestion @schalkneethling! Did a search within the file and replaced all instances (only 4) with `$blue`.